src/index.rst

```diff
@@ -1,28 +1,228 @@
+            fontname="Lato, 'Helvetica Neue', sans-serif",
+            fontsize=12,
+            height=0.1,
+            colorscheme=paired10,
+        ];
+
+        edge [
+            arrowhead=open,
+            arrowsize=0.75,
+        ];
+
+        subgraph Augur {
+            graph [rank=same];
+            node [fillcolor=1, color=2];
+            filter -> align -> tree -> refine -> export;
+        }
+
+        Auspice [fillcolor=3, color=4];
+
+        export -> Auspice;
+
+        subgraph inputs {
+            graph [rank=same];
+            sequences [label="sequences.fasta"];
+            metadata [label="metadata.tsv"];
+        }
+
+        sequences -> filter;
+        metadata -> filter;
+    }
+
+`nextstrain.org <https://nextstrain.org>`__ is a web application to host and
+present the core pathogen builds maintained by the Nextstrain team, as well as
+builds published to :doc:`Nextstrain Groups </guides/share/nextstrain-groups>`
+and :doc:`community pages </guides/share/community-builds>` which are
+maintained and :doc:`shared </guides/share/index>` by many other people.  The
+website incorporates a customized version of Auspice for displaying each
+dataset.
+
+You can run :doc:`Augur <augur:index>` and :doc:`Auspice <auspice:index>` on
+your own computer and use them independently or together with your own builds,
+our core builds, or other's group or community builds.  You can even install
+Auspice on :doc:`your own web server <auspice:server/index>` if you don't want
+to host your builds via nextstrain.org.
+
+The :doc:`Nextstrain CLI <cli:index>` (a program called ``nextstrain``) ties
+together all of the above to provide a consistent way to run pathogen builds,
+access Nextstrain components like Augur and Auspice across computing
+environments such as Docker, Conda, and AWS Batch, and publish datasets to
+nextstrain.org.
+
+:doc:`Nextclade <nextclade:index>` is a web application and two command-line
+programs (``nextclade`` and ``nextalign``) for performing viral genome
+alignment, mutation calling, clade assignment, quality checks, and phylogenetic
+placement.  Nextclade can be used indepedently of other Nextstrain tools as
+well as integrated into builds.
+
+With this overview, you'll be better prepared to dive into the rest of the
+documentation listed in the :ref:`toc` below.
 
-If you have any questions, or simply want to say hi, please give us a shout at hello@nextstrain.org or post at `discussion.nextstrain.org <https://discussion.nextstrain.org>`_.
+If you have questions, need help, or simply want to say hi, please post to our
+`discussion forum <https://discussion.nextstrain.org>`__ where the Nextstrain
+team and other Nextstrain users provide assistance.  For private inquiries,
+please `email the Nextstrain team <hello@nextstrain.org>`__.
 
 .. toctree::
    :maxdepth: 2
    :titlesonly:
-   :caption: Table of contents
+   :caption: Table of Contents
+   :name: toc
 
-   Nextstrain Documentation <self>
```

Comment:
Thank you for removing this! Denisse and I were just discussing how confusing this link is.